format/Message.fbs

```diff
@@ -60,6 +66,14 @@ table RecordBatch {
   /// bitmap and 1 for the values. For struct arrays, there will only be a
   /// single buffer for the validity (nulls) bitmap
   buffers: [Buffer];
+
+  /// If set, each buffer is first compressed with the indicated compressor,
+  /// and then written with the uncompressed length in the first 8 bytes as a
+  /// 64-bit signed integer followed by the compressed buffer bytes (and then
+  /// padding as required by the protocol). The uncompressed length may be set
```

Comment:
Why padding? Are you talking about the 64-byte padding requirement for buffers? That doesn't sound useful if we decompress the buffer anyway.

Reply:
Just the padding requirement of the protocol. I'll remove the comment about padding since it's redundant with the protocol specification (all buffer data must started on an 8-byte aligned offset)